### PR TITLE
feat: fix resubmit

### PR DESCRIPTION
**Pull request by JSHan94**

### CODE REVIEW

File: src/worker/bridgeExecutor/monitor/l1.ts [hunk 1]
```diff
@@ -117,12 +118,12 @@ export class L1Monitor extends Monitor {
...
 
-    const latestTx0 = this.getBlockByHeight(this.latestHeight)?.block.data
-      .txs[0]
+    const latestTx0 = (await this.getBlockByHeight(this.latestHeight))?.block
+      .data.txs[0]
 
     if (!latestTx0) {
       this.logger.info(
-        `[handleNewBlock - ${this.name()}] No txs in height: ${this.latestHeight}`
+        `[handleNewBlock - ${this.name()}] No txs in L2 height: ${this.latestHeight}`
```
Comment (Vritra4) [inline src/worker/bridgeExecutor/monitor/l1.ts:126]: is this correct?

File: src/worker/bridgeExecutor/monitor/resurrector.ts [hunk 2]
```diff
@@ -82,15 +81,41 @@ export class Resurrector {
...
       )
     } catch (err) {
-      if (this.errorCounter++ < 30) {
-        await Bluebird.delay(SECOND)
-        return
+      await this.resubmitFailedDepositTxAtomic(unconfirmedTxs)
+    }
+  }
+
+  async resubmitFailedDepositTxAtomic(
```
Comment (Vritra4) [inline src/worker/bridgeExecutor/monitor/resurrector.ts:88]: is this atomic?

**Review by Vritra4 — APPROVED** [review]
approve anyway

File: src/worker/bridgeExecutor/monitor/resurrector.ts (hunk 2 above)
Comment (JSHan94) [reply src/worker/bridgeExecutor/monitor/resurrector.ts:88]: this is used to send msgs  one by one in failed tx. the name seems not proper, change to `resubmitFailedDepositTxSplit`.

File: src/worker/bridgeExecutor/monitor/l1.ts (hunk 1 above)
Comment (JSHan94) [reply src/worker/bridgeExecutor/monitor/l1.ts]: fixed to `L1 height`.

**Pull request merged**